core/capabilities/ccip/configs/solana/chain_writer.go

```diff
@@ -240,6 +240,7 @@ func getExecuteMethodConfig(fromAddress string, offrampProgramAddress string) ch
 			// are appended to the accounts list in the CCIPExecute args transform for each token transfer
 		},
 		DebugIDLocation: "Info.AbstractReports.Messages.Header.MessageID",
+		// logic receiver and user defined messaging accounts are appended in the CCIPExecute args transform
```

Comment:
Sorry to nit but could you move this line before line 239? Want to keep the remaining account comments in the order that we append them